core/App/ehsm_provider.cpp

```diff
@@ -962,6 +981,18 @@ ehsm_status_t VerifyQuote(ehsm_data_t *quote,
 
     sgx_ql_qe_report_info_t qve_report_info;
     uint8_t nonce[16] = {0};
+    uint8_t mrsigner_quote[SGX_HASH_SIZE] = {0};
+    uint8_t mrenclave_quote[SGX_HASH_SIZE] = {0};
+    uint8_t mrsigner_base_arry[SGX_HASH_SIZE] = {0};
+    uint8_t mrenclave_base_arry[SGX_HASH_SIZE] = {0};
+
+    string mrsigner_base_str(mrsigner_base);
+    string mrenclave_base_str(mrenclave_base);
+
+    for(int i = 0; i < SGX_HASH_SIZE; i++) {
+        mrsigner_base_arry[i] = strtol((mrsigner_base_str.substr(i * 2, 2)).c_str(), NULL, 16);
```

Comment:
Whether need to check ret value here?

Reply:
I think you mean to put this part of the code after checking ret.
Is my understanding correct?

Reply:
I mean the return value of strtol function needs to be checked. I think this function may fail if the input is not a valuable 16-based number or in other scenarios.

Reply:
Ok，I see what you mean. I'll check them